apps/mempool/docker-compose.yml

```diff
@@ -41,6 +41,7 @@ services:
     logging: *default-logging
     restart: on-failure
     stop_grace_period: 1m
+    command: --sql-mode="STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_ENGINE_SUBSTITUTION"
```

Comment:
Maybe this belongs in the Dockerfile not the Docker Compose configuration to keep the concerns separated?

Reply:
I will try to make it works on the Dockerfile :smiley: